TideApp/TideApp/Keys.swift

```diff
@@ -1,12 +0,0 @@
-//
```

Comment:
Unfortunately you can't delete it if you want to make it be built in the CI.
I had just an idea on how to fix this tomorrow, maybe we can just make a build script that create the file with the empty string if its not present!

Reply:
ATM just re-insert it I will do on another branch so we don't mess too much :)